go.mod

```diff
@@ -104,5 +104,7 @@ require (
 
 replace github.com/pingcap/tidb/parser => ./parser
 
+replace github.com/pingcap/kvproto => github.com/3pointer/kvproto v0.0.0-20220311042544-de381a6943c7
```

Comment:
Why replace the default kvproto?

Reply:
need merge this first https://github.com/pingcap/kvproto/pull/881/files